cumulus_library_data_metrics/us_core_v4/documentreference_mandatory.jinja

```diff
@@ -13,7 +13,7 @@ tmp_attachment_flat AS (
         {% if schema["content"]["attachment"] %}
         u.content.attachment.contenttype
         {% else %}
-        NULL AS contenttype
+        CAST(NULL AS VARCHAR) AS contenttype
```

Comment:
afaict, this isn't causing a problem yet, but I caught it in my searching for similar cases